tools/schemacode/bidsschematools/rules.py

```diff
@@ -162,7 +162,11 @@ def _sanitize_extension(ext: str) -> str:
 
 def _stem_rule(rule: bst.types.Namespace):
     stem_regex = re.escape(rule.stem)
-    ext_match = "|".join(_sanitize_extension(ext) for ext in rule.extensions)
+    if isinstance(rule.extensions[0], list) and len(rule.extensions) == 1:
```

Comment:
started trying to fix the failures of schema code but I think I should let grown ups deal with this